### PR TITLE
Start developer version

### PR DESCRIPTION
**Pull request by LeaVerou**
***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/pull/386.html" title="Last updated on Jan 24, 2024, 9:33 AM UTC (6754594)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/design-principles/386/7627102...6754594.html" title="Last updated on Jan 24, 2024, 9:33 AM UTC (6754594)">Diff</a>

### CODE REVIEW

File: dev/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+To generate developer version `cd` to the root of this repo, then run:
```
Comment (hober) [inline dev/README.md:1]: The Makefile should be updated to have a rule that does this, and these instructions should simply say something like "cd to the root of this repo and type `make dev`" or something.

**Review by hober — CHANGES_REQUESTED** [review]
Please [update the Makefile](https://github.com/w3ctag/design-principles/pull/386/files#r1172101424)

File: dev/README.md (hunk 1 above)
Comment (plinss) [reply dev/README.md:1]: Given that we're using Bikeshed, which is Python based, I'd rather not add a nodejs dependency.

**Comment by LeaVerou** [comment]
Per breakout discussion today: 

- [x] Action item for @LeaVerou to ping @tabatkins to fix Bikeshed bug
- [ ] Action item for @plinss to explore using Jinja + make to produce the two versions, so we don't depend on Bikeshed. Then we can also have more fine grained conditionals, whereas bikeshed can only do whole elements, so it may be a better solution altogether.

File: index.bs [hunk 2]
```diff
@@ -1035,7 +1046,7 @@ and because the <{img}> element is an empty element that does not permit child e
 A space-separated syntax would not have been possible, as each list item includes multiple values.
 
 <!--
-TODO: link to the "microsyntaxes are usually bad" principle once it's been written. 
+TODO: link to the "microsyntaxes are usually bad" principle once it's been written.
```
Comment (cynthia) [inline index.bs:1049]:
```suggestion
TODO: link to the "microsyntaxes are usually bad" principle once it's been written. 
```

**Comment by LeaVerou** [comment]
Closing this since we resolved to use a Python-based workflow rather than Bikeshed's inclusion/exclusion capabilities. @plinss you said you'd explore using Jinja to preprocess the document, did you make any progress on that? If are able to just add it to the spec, we can take it from there.

**Pull request closed without merging**